### DISCUSSION TITLE
Using optional dependencies

### DISCUSSION THREAD

**Discussion by qpre** [vercel/turborepo#1287] [Help] [answered]
Hello,

We're trying to migrate one of our projects to turborepo but it includes optionalDependencies,
before moving to turborepo we were installing via `yarn install --ignore-optional`
but that can't be done with turbo as its installation relies on optionalDependencies not being ignored.

how can one work around this ?

Thanks !

**Comment by villanuevadani** [comment]
Any update on this?

**Comment by anthonyshew** [comment, marked as answer]
Could you please open an issue if this is still a problem for you? 🙏
